lib/CL/devices/tce/tce_common.cc

```diff
@@ -71,11 +71,14 @@ TCEDevice::TCEDevice(cl_device_id dev, const char* adfName) :
   POCL_INIT_LOCK(tce_compile_lock);
   dev->address_bits = 32;
   dev->autolocals_to_args = 1;
+  /* This assumes TCE is always Big-endian;
```

Comment:
Please mirror this assumption now as we consider big endian TCE almost like in deprecated status now. All new TTAs are LE basically.